### DISCUSSION TITLE
Add ERC-721 on the Explorer

### DISCUSSION THREAD

**Discussion by samui68400** [zkSync-Community-Hub/zksync-developers#555] [Feature requests]
### Network

zkSync Era

### Provide a brief description of the feature you would like to see implemented.

Implement a feature in the zkSync block explorer to list all ERC-721 tokens, similar to Taiko's explorer. 

**This would include details such as Holder Count and Transfer Count for each token.**

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

This feature would benefit users, developers, and the protocol by providing easy access to important information about ERC-721 tokens on zkSync. 

**It enhances transparency and usability, making it easier to track and interact with NFTs on the network.**

### Additional Details

Display token name, address, transfer count, total supply, and holder count.
<img width="1432" alt="Capture d’écran 2024-05-31 à 08 17 22" src="https://github.com/zkSync-Community-Hub/zksync-developers/assets/107585158/e13a805b-0871-473d-9d95-2c3b499f9e9c">

- Include filtering and sorting options for better user experience.
- Ensure real-time updates to maintain accuracy.
- This implementation would align zkSync with competitor networks and improve overall user engagement and satisfaction.

**Comment by bxpana** [comment]
Thank you for the request. This is definitely something we know the community wants and something that we are planning to implement in the future.

**Reply by bxpana** [reply]
If you know more people interested in this, it would be great to get this post upvoted so we can see the demand from the community.

**Reply by samui68400** [reply]
I will try to make people upvote that

**Comment by Praizl** [comment]
This would be a useful feature. 
I think zksync would soon have an explosion in the number of tokens built on the L2, and having access to these features will reduce the need to use third-party dapps to check those metrics
